src/main/resources/templates/search.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Use `<html lang="en" xmlns:th="https://www.thymeleaf.org">` here to imply the Thymeleaf XML namespace.